ghost/core/core/server/services/auth/api-key/admin.js

```diff
@@ -60,7 +61,7 @@ const authenticate = function apiKeyAdminAuth(req, res, next) {
         }));
     }
 
-    return authenticateWithToken(req, res, next, {token, JWT_OPTIONS: JWT_OPTIONS_DEFAULTS});
```

Comment:
RE: JWT_OPTIONS_DEFAULTS - do we have sufficient test coverage to make sure this change doesn't break anything? I saw the `ignoreMaxAge` being passed around explicity now. 

Ideally this type of change should land in `main` and then we'd rebase over it, because we have to idea when this `arch` branch will get merged and the change would be good to validate individually, not as a part of this big refactor 🤔

Reply:
It's only used in these two functions, and one of them omits the `maxAge` option. To me this is a small enough change to see that the logic is the same. I can look at adding some more tests if you disagree tho

Agree that we shouldn't put this change into `main` as part of a huge refactor, but not sure if _now_ is the right time to get it into `main`?